core/include/ldml/keyboardprocessor_ldml.ts

```diff
@@ -245,6 +245,60 @@ class Constants {
    */
   readonly keys_flags_extend = 1;
 
+  /**
```

Comment:
Note that the `keys_mod_alt` and `keys_mod_ctrl` masks do not currently correspond to the KMX modifier bitmasks in kmx_file.h. Perhaps we should make them match? KMX distinguishes between Alt and L/R Alt -- each gets a separate bit in the mask. This is partly historical (originally kmx did not support L/R Alt).

https://github.com/keymanapp/keyman/blob/ebff42e657fcf8bd9f32a36b1f1d7114903a4cd3/common/include/kmx_file.h#L268-L289

```suggestion
  /* Modifier key bitmasks. These correspond to the modifier bitmasks from kmx_file.h */
  
  /**
```

Reply:
@mcdurdin  `alt` and `ctrl` as such are needed to support the spec. `alt` should be `0x4 | 0x8 === 0xC`

Reply:
Yeah, we can diverge from kmx on the `alt` and `ctrl` modifiers -- just should note that we are doing that.